scripts/ci/run_ci.sh

```diff
@@ -122,33 +122,38 @@ function handle_coverage() {
 	echo "Calling gcovr"
 	gcovr -r ${ZEPHYR_BASE} -x > shippable/codecoverage/coverage.xml;
 
-	# Capture data
-	echo "Running lcov --capture ..."
-	lcov --capture \
-		--directory sanity-out/native_posix/ \
-		--directory sanity-out/nrf52_bsim/ \
-		--directory sanity-out/unit_testing/ \
-		--directory bsim_bt_out/ \
-		--output-file lcov.pre.info -q --rc lcov_branch_coverage=1;
-
-	# Remove noise
-	echo "Exclude data from coverage report..."
-	lcov -q \
-		--remove lcov.pre.info mylib.c \
-		--remove lcov.pre.info tests/\* \
-		--remove lcov.pre.info samples/\* \
-		--remove lcov.pre.info ext/\* \
-		--remove lcov.pre.info *generated* \
-		-o lcov.info --rc lcov_branch_coverage=1;
-
-	# Cleanup
-	rm lcov.pre.info;
-	rm -rf sanity-out out-2nd-pass;
```

Comment:
this line should be below the if

Reply:
copied it after the conditional to make sure it happens in any case.